run-basic-batch-operators-test.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/sh
+
+# Test run timestamp
+TIMESTAMP=$(date | sed 's/[ :]/_/g')
```

Comment:
The timestamp format is not really timestamp.
```suggestion
TIMESTAMP=$(date +%Y%m%d-%H:%M:%S)
```

Reply:
fixed